MetricsDashboard.WebApi/appsettings.json

```diff
@@ -9,7 +9,6 @@
   "AllowedHosts": "*",
   "Application": {
     "MongoConnectionString": "mongodb://localhost:27017",
```

Comment:
You could rename MongoConnectionString to ConnectionString and remove property in Settings class